docs/man/grcli-events.1.md

```
@@ -0,0 +1,29 @@
+GRCLI-EVENTS 1 @DATE@ "grout @VERSION@"
+========================================
+
+# NAME
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)